lib/node_modules/@stdlib/stats/base/meanpn/examples/index.js

```diff
@@ -18,18 +18,12 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
-var Float64Array = require( '@stdlib/array/float64' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
 var meanpn = require( './../lib' );
 
-var x;
-var i;
-
-x = new Float64Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-	x[ i ] = round( (randu()*100.0) - 50.0 );
-}
+var x = discreteUniform( 10, 0, 100, {
```

Comment:
```suggestion
var x = discreteUniform( 10, -50, 50, {
```

Preserve the original value range. Also, you need to update the example in the README, as well, to match the changes you made here.